apps/adminer.sh

```diff
@@ -18,16 +16,46 @@ debug_mode
 # Check if root
 root_check
 
-# Warn user about HTTP/2
-http2_warn Adminer
-
 # Check that the script can see the external IP (apache fails otherwise)
+check_external_ip() {
 if [ -z "$WANIP4" ]
 then
     print_text_in_color "$IRed" "WANIP4 is an emtpy value, Apache will fail on reboot due to this. Please check your network and try again."
     sleep 3
     exit 1
 fi
+}
+
+# Check if adminer is already installed
+print_text_in_color "$ICyan" "Checking if Adminer is already installed..."
+if is_this_installed adminer
+then
+    msg_box "It seems like 'Adminer' is already installed.\nIf you continue, Adminer and all Adminer-settings get deleted."
+    if [[ "no" == $(ask_yes_or_no "Do you really want to continue?") ]]
+    then
+        exit
+    fi
+    check_external_ip
+    print_text_in_color "$ICyan" "Uninstalling Adminer and resetting all settings..."
+    a2disconf adminer.conf
+    rm $ADMINER_CONF
+    rm $ADMINERDIR/adminer.php
+    check_command apt purge adminer -y
+    if ! restart_webserver
+    then
+        msg_box "Apache2 could not restart...\nThe script will exit."
+        exit
+    fi
+    msg_box "Adminer was successfully uninstalled and all settings were resetted."
+    exit
+fi
+
+print_text_in_color "$ICyan" "Installing and securing Adminer..."
+
+# Warn user about HTTP/2
+http2_warn Adminer
+
+check_external_ip
```

Comment:
Add comment?

Reply:
ping @szaimen Why isn't check_external_ip in lib.sh, and are there more occurrences like this that would need to be replaced?

Reply:
@szaimen Can you answer this please?

Reply:
It was before a script, that I have made a function from, because I need it two times in the script. I will put it into the lib.sh

Reply:
and no, there are not more occurentes like this afaik.

Reply:
this should be done.